images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
@rmohr I do not sure that go binding for libvirt will check auth.conf at all, because we use `libvirt.NewConnectWithAuth` in our `newConnection` method and this method assume that you will give to it user and password